content/documentation/4-batch-developer-guides/2-batch/3-data-flow-simple-task.md

```diff
@@ -1,32 +1,101 @@
 ---
 path: 'batch-developer-guides/batch/data-flow-simple-task/'
-title: 'Launch a Spring Cloud Task application using Data Flow'
-description: 'Launch a Spring Cloud Task application using Data Flow'
+title: 'Register and Launch a Spring Cloud Task application using Data Flow'
+description: 'Register and Launch a Spring Cloud Task application using Data Flow'
 ---
 
 # Deploy a Spring Cloud Task application using Data Flow
 
-In this section we will show how to register task applications with Data Flow, create a Task DSL, and deploy the task to Cloud Foundry, Kubernetes and your local machine.
+In this section, we will demonstrate how to register a Spring Cloud Task application with Data Flow, create a task definition, and launch the task on Cloud Foundry, Kubernetes and your local machine.
 
 ## Development
```

Comment:
I will make one change on this title.  From "Development" to "Create Task Definition"